.github/CODEOWNERS

```diff
@@ -120,6 +120,7 @@ build-utils/        @getsentry/owners-js-build
 /src/sentry/api/endpoints/organization_event_details.py         @getsentry/visibility
 /src/sentry/api/endpoints/organization_events.py                @getsentry/visibility
 /src/sentry/api/endpoints/organization_events_facets.py         @getsentry/visibility
+/src/sentry/api/endpoints/organization_events_facets_performance.py         @getsentry/visibility
```

Comment:
🤔  do we care about alignment in this file?

Reply:
as long as it works that's good enough for me